src/scales/period.js

```diff
@@ -25,11 +25,17 @@ export class PeriodScale extends BaseScale {
 
         var periodGenerator = UnitDomainPeriodGenerator.get(props.period);
         if (props.fitToFrameByDims || (periodGenerator === null)) {
-            this.vars = _(vars).chain()
-                .uniq((x) => new Date(x).getTime())
+            this.vars = _.unique(vars)
```

Comment:
@juanjoDiaz

Should be

```
_.unique(vars, (x) => new Date(x).getTime())
```

We can't skip unique iterator here since _.unique compare dates by reference (not date value).

```
var a = new Date('2016-01-01');
var b = new Date('2016-01-01');
_.unique([a, b]) // returns 2 items with equal values

// vs

_.unique([a, b], (x) => new Date(x).getTime()) // returns 1 item
```